### ISSUE TITLE
Module AbstractModelMethods no longer does anything

### ISSUE THREAD

**Issue by prioux** [aces/cbrain#822]
That module use to flag classes and prevent new() from working, but it might have been too harsh in the past. The code for new() is commented out. It should still do its job of preventing objects from abstract classes from being saved, at least. To fix.

E.g. in the console

```
g=Group.new
g.type = 'Group'
g.name = "aaa"
g.save! # this works, but shouldn't.
```

**Comment by prioux** [comment]
Also, it seems it was trying to store its flag as a class instance variable, which will not work because subclasses will see it too... ?

**Comment by prioux** [comment]
Another test in the console:

```ruby
r=RemoteResource.new(:name => 'aaa', :user_id => 1, :group_id =>1)
r.save!
```

This worked before, but it shouldn't anymore.

**Comment by prioux** [comment]
Fixed by #823
